lib/affirm/charge_event.rb

```diff
@@ -1,20 +1,20 @@
 module Affirm
   class ChargeEvent
     attr_reader :id, :transaction_id, :type, :created,
-      :amount, :fee, :order_id, :shipping_carrier, :shipping_confirmation,
-      :fee_refunded
+                :amount, :fee, :order_id, :shipping_carrier, :shipping_confirmation,
```

Comment:
Could this and other spacing/formatting changes be removed? They're not relevant to the PR.